config/environments/development.rb

```diff
@@ -47,17 +49,25 @@
 
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
 
   # Suppress logger output for asset requests.
   config.assets.quiet = true
   config.active_record.verbose_query_logs = true
+  # Tell Active Support which deprecation messages to disallow.
```

Comment:
Would it be possible to put an empty line between each configuration and the next comment?  This should make it clearer which configuration each comment relates to (and maintain consistency with the other configs).